src/pages/gaps/index.tsx

```diff
@@ -116,12 +116,14 @@ const GapsPage = () => {
 
   return (
     <PageContainer>
-      <Typography className="page-title" variant="h4">
+      <Typography variant="h4" className="page-title">
         {t('gaps_page_title')}
+        <InfoYoutubeModal
+          label={t('open_video_about_this_page')}
+          title={t('gaps_page_description')}
+          videoUrl=""
```

Comment:
the video url is empty

Reply:
It is empty because as far as I have seen there is no suitable video to put in place. I thought that maybe down the road someone could produce a suitable video.
If you have another idea I would be happy, or even better if there is a suitable video that I missed.

I also thought of adding it so that it was clear, write me if you want to add it
![Screenshot 2024-06-11 123406](https://github.com/hasadna/open-bus-map-search/assets/100128949/f11b1026-5c10-4640-b6a4-90052f88a2d7)

the PR is ready for review.

Reply:
thanks for the clarification!
so I guess we should / could remove the iframe if there's no video to be shown (?)

Reply:
I don't want to destroy the element
And thank you very much for the feedback

Reply:
what do you mean by destring the element?
I meant to say that if the modal component gets no url,  it should not render an iframe inside of it